compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -480,6 +480,7 @@ impl<'a> Resolver<'a> {
                         "const parameters may only be used as standalone arguments, i.e. `{}`",
                         name
                     ));
+                    err.note("use feature(const_generics) and feature(const_evaluatable_checked) to enable this");
```

Comment:
```suggestion
                    err.note("use `#[feature(const_generics)]` and `#![feature(const_evaluatable_checked)]` to enable this");
```
we tend to already use `#![feature(feature_name)]` in other error messages, so let's stay consistent here.

---
You only emit the `note` for const parameters, but the same holds true of type params as well.

Consider putting it note after the `if`-statement.
---
I feel like "to enable this" doesn't fit too well because the rest of this error talks about a restriction.
Maybe "to loosen this restriction" or "to allow generic const expressions"?

Reply:
Alright, I didn't like `to enable this` either but I couldn't think of something I liked better.

Will change and the others too